### PR TITLE
chore: update folder

### PR DESCRIPTION
**Pull request by SirXuNian**
fix SkinMeshRender not work.
add gltf loader test sample
add gltf parser
Removes all non-ASCII characters
update folder fix not run

### CODE REVIEW

File: sample/Sample_base_0.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { Engine3D } from "../src/Engine3D";
```
Comment (lslzl3000) [inline sample/Sample_base_0.ts:1]: better import all modules from index.ts, just like import all from '@orillusion/core'

File: sample/Sample_base_0.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        this.hoverCameraController = mainCamera.object3D.addComponent(HoverCameraController);
+        this.hoverCameraController.setCamera(15, -15, 10);
+
+        await this.initScene(this.scene);
+
+        let view = new View3D();
+        view.scene = this.scene;
+        view.camera = mainCamera;
+
+        // renderJob.addPost(new SSAOPost());
```
Comment (lslzl3000) [inline sample/Sample_base_0.ts:52]: better clean all demos, remove unused lines

File: sample/Sample_base_0.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            lc.intensity = 1.7;
+            scene.addChild(this.lightObj);
+        }
+
+        {
+        }
+        return true;
+    }
+
+    loop() { }
```
Comment (lslzl3000) [inline sample/Sample_base_0.ts:95]: no need add loop

File: sample/animation/Sample_Skeleton.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        view.camera = mainCamera;
+
+        await this.initScene(this.scene);
+
+        Engine3D.startRenderView(view);
+    }
+
+    async initScene(scene: Scene3D) {
+        {
+            let root = await Engine3D.res.loadGltf('sample/animation/assets/CesiumMan/CesiumMan_compress.gltf');
```
Comment (lslzl3000) [inline sample/animation/Sample_Skeleton.ts:51]: use cdn url? or add assets submodule

File: sample/animation/assets/CesiumMan/CesiumMan_compress.bin
Comment (lslzl3000) [inline sample/animation/assets/CesiumMan/CesiumMan_compress.bin:1]: should not add these files in git repo

File: src/assets/Res.ts [hunk 5]
```diff
@@ -288,4 +294,100 @@ export class Res {
             .catch((e) => {
             });
     }
+
```
Comment (lslzl3000) [inline src/assets/Res.ts:297]: empty lines?

**Comment by lslzl3000** [comment]
We may not merge any samples into this repo before they are ready, just keep them in local

**Review by hellmor — APPROVED** [review]

**Pull request merged**